osgp/protocol-adapter-oslp/web-device-simulator/src/main/java/org/opensmartgridplatform/webdevicesimulator/service/OslpChannelHandler.java

```diff
@@ -494,6 +495,21 @@ private void sleep(final Long sleepTime) {
             }
         }
 
+        response = this.handleRequest(request, response, device);
```

Comment:
as far as I can see the parameter response is always null here.

Reply:
Removed the response parameter.